### PR TITLE
add new company - Capmo

### PR DESCRIPTION
**Pull request by joaocarmo**
This is a modified version of, and should adhere to, the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

* [ ] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply**
* [x] You are an employee of the company mentioned and confirm all included details are correct.
* [x] You know your alphabet - company is listed in alphabetical order in the README.md file.
* [x] The company directly hires employees. ** No bootcamps / freelance sites / etc**
* [x] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
* [x] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
* [x] An entry has been added or edited within the [readme.md file](../../main/README.md).
* [x] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
* [x] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
* [x] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -775,4 +776,4 @@ Name | Website | Region
 [Zignaly.com](/company-profiles/zignaly.md) | https://zignaly.io/ | Worldwide
 [Zip.co](/company-profiles/zip-co.md) | https://zip.co/us/ | USA, Canada
 [Zolar](/company-profiles/zolar.md) | https://www.zolar.de/ | Germany
-[Zup](/company-profiles/zup.md) | https://zup.com.br/ | Brazil
\ No newline at end of file
+[Zup](/company-profiles/zup.md) | https://zup.com.br/ | Brazil
```
Comment (joaocarmo) [inline README.md:779]: **note:** this is a whitespace change. my linter probably _fixed it_.

**Comment by joaocarmo** [comment]
> @joaocarmo
> 
> The site build will complain about the company title vs the name in the readme. Either the readme needs to show `Capmo GmbH` or the title in the company profile needs to be `Capmo` - i.e. they need to match and the company profile filename then needs to match. So
> 
> Capmo and `capmo.md` or Capmo GmbH and `capmo-gmbh.md`

Good point! I wasn't aware of that. I believe the simplest change is to simplify the company profile name to just `Capmo`. Fixed as suggested.

**Review by dougaitken — APPROVED** [review]
I'm not wild about using timezones but I'll allow it since I don't have a good alternative.

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated @joaocarmo

**Pull request merged**